.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @braqzen
+* @nfurfaro
```

Comment:
```suggestion
* @FuelLabs/application-dev 
```

Reply:
I think making the team the owners of the code could work.

Reply:
I'm happy to approve if they accept your commit suggestion.